js/src/product-feed/product-statistics/index.js

```diff
@@ -133,6 +134,7 @@ const ProductStatistics = () => {
 					</SummaryList>
 				) }
 			</CardBody>
+			<CreateCampaignNotice />
```

Comment:
It could have been, but doing so caused layout conflicts with the existing `Flex` container. I think changing the component to use a `Flex` wrapper instead of a `Notice` could help here, so we'll give it a try.

Reply:
I think this looks better after [165543c](https://github.com/woocommerce/google-listings-and-ads/pull/2641/commits/165543cc1f6e87d45b75660203e1c081476335f0).

<img width="1041" alt="image" src="https://github.com/user-attachments/assets/c775a8dc-6893-4eff-b7f3-2f16eebefb02">